docs/src/ReactPlayground.js

```diff
@@ -6,6 +6,8 @@ const classNames = require('classnames');
 const React = require('react');
 const ReactDOM = require('react-dom');
 
+const bootstrapUtils = require('../../src/utils/bootstrapUtils.js').default;
```

Comment:
No need (:

We need to first figure out how we want to fix the exports from `bootstrapUtils`, since there's two ways to go that produce identical results for CJS users, but different ones for users of polyfilled ES2015 modules.

Thanks for these docs, BTW!